src/Microsoft.DotNet.Interactive.Documents/InteractiveDocument.cs

```diff
@@ -264,19 +259,23 @@ internal static void MergeKernelInfos(KernelInfoCollection destination, KernelIn
         destination.AddRange(source.Where(ki => added.Add(ki.Name)));
     }
 
-    internal static bool TryGetKernelInfoFromMetadata(
+    public bool TryGetKernelInfosFromMetadata(
+        [NotNullWhen(true)] out KernelInfoCollection? kernelInfos) =>
+        TryGetKernelInfosFromMetadata(Metadata, out kernelInfos);
+
+    internal static bool TryGetKernelInfosFromMetadata(
         IDictionary<string, object>? metadata,
-        [NotNullWhen(true)] out KernelInfoCollection? kernelInfo)
+        [NotNullWhen(true)] out KernelInfoCollection? kernelInfos)
```

Comment:
Can this be inlined into the above non-static overload (with all callers switched to calling the non-static overload)? Or can this be made private?

Please ignore the above if we have more callers for the static overload outside the current class.